features/specs/gezag-minderjarige/verblijf-buitenland.feature

```diff
@@ -0,0 +1,280 @@
+        | ouder 'Aart' en een derde  | gezamenlijk gezag met ouder 'Aart' en een onbekende derde     |
+
+    Voorbeeld: Er is in een gerechtelijke uitspraak gezag toegewezen en daarna is de minderjarige geëmigreerd
+      Gegeven persoon 'Bert'
+      * heeft 'Aart' en 'Gerda' als ouders
+      En er is 2 jaar geleden een gerechtelijke uitspraak over het gezag geweest waarbij het gezag over 'Bert' is toegewezen aan <toegewezen aan>
+      En 'Bert' is 1 jaar geleden geëmigreerd naar België
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' <gezag uitspraak>
+
+      Voorbeelden:
+        | toegewezen aan             | gezag uitspraak                                               |
+        | 'Gerda'                    | eenhoofdig ouderlijk gezag met ouder 'Gerda'                  |
+        | 'Aart'                     | eenhoofdig ouderlijk gezag met ouder 'Aart'                   |
+        | beide ouders               | gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart' |
+        | een voogdijinstelling      | voogdij                                                       |
+        | ouder 'Gerda' en een derde | gezamenlijk gezag met ouder 'Gerda' en een onbekende derde    |
+        | ouder 'Aart' en een derde  | gezamenlijk gezag met ouder 'Aart' en een onbekende derde     |
+
+    Voorbeeld: Er is een gerechtelijke uitspraak gezag toegewezen, daarna is de minderjarige geëmigreerd en vervolgens weer geïmmigreerd
+      Gegeven persoon 'Bert'
+      * heeft 'Aart' en 'Gerda' als ouders
+      En er is 2 jaar geleden een gerechtelijke uitspraak over het gezag geweest waarbij het gezag over 'Bert' is toegewezen aan <toegewezen aan>
+      En 'Bert' is 1 jaar geleden geëmigreerd naar België
+      En 'Bert' is vorige maand geïmmigreerd naar Nederland
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' <gezag uitspraak>
+
+      Voorbeelden:
+        | toegewezen aan             | gezag uitspraak                                               |
+        | 'Gerda'                    | eenhoofdig ouderlijk gezag met ouder 'Gerda'                  |
+        | 'Aart'                     | eenhoofdig ouderlijk gezag met ouder 'Aart'                   |
+        | beide ouders               | gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart' |
+        | een voogdijinstelling      | voogdij                                                       |
+        | ouder 'Gerda' en een derde | gezamenlijk gezag met ouder 'Gerda' en een onbekende derde    |
+        | ouder 'Aart' en een derde  | gezamenlijk gezag met ouder 'Aart' en een onbekende derde     |
+
+  Regel: Het gezag kan worden bepaald voor een minderjarige die ooit Nederland als vaste verblijfplaats heeft gehad en waarvoor van rechtswege gezamenlijk ouderlijk gezag is bepaald
+
+    Voorbeeld: minderjarige is in het buitenland geboren en heeft twee ouders die met elkaar gehuwd zijn
```

Comment:
in alle voorbeelden geldt: is het belangrijk dat minderjarige staande huwelijk geboren is (zie opmerking H. Zonneveld)